### PR TITLE
ILLink Perf: Fix Roslyn benchmark

### PR DESCRIPTION
**Pull request by swaroop-sridhar**
Use ILLink.Tasks package to build the Roslyn linked benchmark,
now that Roslyn can target netcoreapp2.0.

### CODE REVIEW

**Comment by swaroop-sridhar** [comment]
@dotnet-bot help

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@sbomer please review

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot  test 
Windows_NT x86 full_opt legacy_backend CoreCLR Perf Tests Correctness

File: tests/src/performance/linkbench/scripts/build.cmd [hunk 1]
```diff
@@ -119,42 +119,12 @@ exit /b
...
-
-REM Run Linker
-echo Running %__dotnet1% %__IlLinkDll% -t -c link -a @%AssetDir%\Roslyn\RoslynRoots.txt -x %AssetDir%\Roslyn\RoslynRoots.xml -l none -out ..\Linked
-call %__dotnet1% %__IlLinkDll% -t -c link -a @%AssetDir%\Roslyn\RoslynRoots.txt -x %AssetDir%\Roslyn\RoslynRoots.xml -l none -out ..\Linked
-if errorlevel 1 set ExitCode=1&& echo Roslyn: ILLink failed
-
+pushd %LinkBenchRoot%\roslyn\src\Compilers\CSharp\csc
+call %__dotnet2% restore -r win10-x64
+call %__dotnet2% publish -c release -r win10-x64 -f netcoreapp2.0 /p:LinkDuringPublish=false --output ..\..\..\..\Binaries\release\Exes\csc\netcoreapp2.0\win10-x64\Unlinked
+if errorlevel 1 set ExitCode=1&&echo HelloWorld: publish failed
```
Comment (sbomer) [inline tests/src/performance/linkbench/scripts/build.cmd]: Should say Roslyn, not HelloWorld.

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Comment by swaroop-sridhar** [comment]
@dotnet-bot test Windows_NT x64 illink

**Pull request merged**